### ISSUE TITLE
What does each column mean in the annotated data?

### ISSUE THREAD

**Issue by jojonki** [xliuhw/NLU-Evaluation-Data#1]
First of all, this project is amazing. And the results reported in your paper is quite interesting.

Now, I am trying to use your annotated data. But there are no documents about the header.
```
userid;answerid;scenario;intent;status;answer_annotation;notes;suggested_entities;answer_normalised;answer;question;ml_tier1_int_type;ml_tier1_int_res;ml_tier1_int_f1;ml_tier1_ent_res;ml_tier2_int_type;ml_tier2_int_res;ml_tier2_int_f1;ml_tier2_ent_res;ml_tier3_int_type;ml_tier3_int_res;ml_tier3_int_f1;ml_tier3_ent_res;rule_int_type;rule_int_res;rule_ent_res;rule_covered
```

Especially among them, I could not understand the below elements. Could you teach me the details?
- status
- notes
- ml_tier_xxx
- rule_xxx

**Comment by xliuhw** [comment]
Thanks again for the comments!

For a quick answer, "status" is for our script to generate the training/test dataset. It is either "null" or something starting with "IRR_" which means "irrelavent" and added by the annotators or the developers. The script will skip these utterance when generating the trainning/test set.

The purpose for this is that we didn't want to delete any lines of the originally collected the data so it would be helpful for checking the consistence with other processes in our processing pipelines.

The "notes" was purely for monitoring the annotation processes, to see if something has been changed by the annotators.

The others like "ml_tier_xxx" and "rule_xxx" were for our internal tests purpose. I though I had removed them from the published one but somehow they are still there. Sorry, I will tide it up soon later on.
